ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -67,6 +71,35 @@ def __init__(self, tracer=None, config=None, service=None):
         except Exception:
             log.warning("Failed to load CODEOWNERS", exc_info=True)
 
+    def _configure_writer(self):
+        writer = None
+        if ddconfig._ci_visibility_agentless_enabled:
+            site_id = os.environ.get("DD_SITE")
```

Comment:
https://github.com/DataDog/dd-trace-py/pull/5497#discussion_r1166445998 I think we need to check for `DD_API_KEY` here and not `DD_SITE`?

Reply:
Good catch, thanks!